core/commons/che-core-commons-test/pom.xml

```diff
@@ -21,10 +21,30 @@
     <artifactId>che-core-commons-test</artifactId>
     <packaging>jar</packaging>
     <name>Che Core :: Commons :: Utilities for tests</name>
+    <properties>
+        <findbugs.failonerror>true</findbugs.failonerror>
+    </properties>
     <dependencies>
         <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
```

Comment:
Yes i understand, i tried to explain why the guava scope is _provided_, can you please explain why api should use dependencies with provided scope, afaik we don't use such practice